src/pages/Explorer/Category/Widget.svelte

```diff
@@ -1,53 +1,37 @@
 <script>
   import Svg from 'webkit/ui/Svg/svelte'
   import Category from './Category.svelte'
-  import ChartLayoutItem from '../Layouts/ChartLayoutItem.svelte'
-  import SocialTrend from '../Layouts/SocialTrend.svelte'
-  import WeeklyReport from '../Layouts/WeeklyReport.svelte'
-  import SheetsTemplate from '../Layouts/SheetsTemplate.svelte'
 
   export let title = 'Recent Chart Layouts'
   export let icon = 'info'
   export let color = 'green'
   export let iconWidth = 16
-  export let type
-  export let items = [
-    {
-      id: 0,
-      type: 'chart',
-      title: 'Protocol - Fundamental comparison (TVL, User, Develel)',
-      comments: 0,
-      votes: 0,
-      user: { username: 'test' },
-      assets: [
-        { slug: 'bitcoin' },
-        { slug: 'ethereum' },
-        { slug: 'bitcoin-cash' },
-        { slug: 'bitcoin' },
-      ],
-    },
-  ]
+  export let getItems = (page) => new Promise((resolve) => resolve([]))
+
+  let items = []
+  let page = 1
+  let pages = 1
+
+  $: hasMore = pages > 1 && page < pages
+  $: page, getItems().then((res) => (items = res))
+
+  function onMore() {
+    if (page < pages) {
+      page += 1
+    }
+  }
 </script>
 
-<Category {title} small {items}>
+<Category small {title} {items} {hasMore} {onMore}>
   <div
     slot="icon"
     style="fill:var(--{color}); background:var(--{color}-light-1)"
-    class="$style.icon row hv-center"
-  >
+    class="$style.icon row hv-center">
     <Svg id={icon} w={iconWidth} />
   </div>
 
   <svelte:fragment let:item>
-    {#if type === 'social'}
-      <SocialTrend {item} />
-    {:else if type === 'weekly_report'}
-      <WeeklyReport {item} />
-    {:else if type === 'sheets_templates'}
-      <SheetsTemplate {item} />
-    {:else}
-      <ChartLayoutItem small {item} />
-    {/if}
+    <slot name="item" {item} />
```

Comment:
Better to make it a default slot, since it's the most important composition part
```suggestion
    <slot {item} />
```